executor/prepared.go

```diff
@@ -177,14 +176,14 @@ func (e *PrepareExec) Next(ctx context.Context, req *chunk.Chunk) error {
 		Stmt:          stmt,
 		StmtType:      GetStmtLabel(stmt),
 		Params:        sorter.markers,
-		SchemaVersion: e.is.SchemaMetaVersion(),
+		SchemaVersion: ret.InfoSchema.SchemaMetaVersion(),
```

Comment:
Actually the `infoschema` should keep unchanged afte the change?

Reply:
Stale reads mean to read things at a specific time. So if we will support prepare stale select eventually, it should be unchanged.